lib/ansible/modules/files/acl.py

```diff
@@ -99,14 +96,14 @@
     permissions: r
     state: present
 
-- name: Removes the acl for Joe on a specific file
+- name: Removes the ACL for Joe on a specific file
   acl:
     path: /etc/foo.conf
     entity: joe
     etype: user
     state: absent
 
-- name: Sets default acl for joe on foo.d
+- name: Sets default ACL for joe on foo.d/
```

Comment:
remove stray / at the end...

Reply:
Well, without the trailing /, it is unclear this is a directory.
I think it is good practice to keep a trailing slash for directories (even when they are strictly no needed).